src/util/IfOverflowMatches.ts

```diff
@@ -1,4 +1,6 @@
-export const ifOverflowMatches = (props: any, value: string) => {
+export type IfOverflow = 'hidden' | 'visible' | 'discard' | 'extendDomain';
```

Comment:
Not a blocker:

If we add a comment above the type, that would then show up in storybook as a description and become the user facing explanation as to what this does.